Formula/airspy.rb

```diff
@@ -1,5 +1,5 @@
 class Airspy < Formula
-  desc "The usemode driver and associated tools for airspy"
+  desc "Tiny and efficient software defined radio"
```

Comment:
```suggestion
  desc "Driver and tools for the software-defined Airspy radio"
```

The formula contains the driver and tools rather than the radio